addons/zylann.hterrain/tools/plugin.gd

```diff
@@ -86,6 +88,10 @@ var _mouse_pressed := false
 #var _pending_paint_action = null
 var _pending_paint_commit := false
 
+var _overlay_selector : HT_BrushEditorOverlay = null
+var _widget_action_name : String = ""
```

Comment:
Been thinking, I don't see a reason for this to be a string. It could just be an enum integer. Or maybe not even exist?

Reply:
You are right. For what we have now, it doesn't.  I've removed it.